client/verta/verta/_registry/modelversion.py

```diff
@@ -208,6 +208,9 @@ def get_artifact(self, key):
         return artifact_stream
 
     def del_artifact(self, key):
+        if key == "model":
+            raise ValueError("the key \"model\" is reserved for model; consider using del_model() instead")
```

Comment:
I think this sounds a little more straightforward than "key is reserved" for deletion
```suggestion
            raise ValueError("model can't be deleted through del_artifact(); consider using del_model() instead")
```

Reply:
Fixed.